src/file.js

```diff
@@ -1,38 +1,19 @@
 import fs from 'fs'
-import gulp from 'gulp'
 import path from 'path'
 import glob from 'glob'
-import rimraf from 'rimraf'
-import cache from 'gulp-cached'
-import gwatch from 'gulp-watch'
 import chokidar from 'chokidar'
 import archiver from 'archiver'
 import {Promise, promisify} from 'bluebird'
 
 const mkdir = promisify(fs.mkdir)
-const bbRimraf = promisify(rimraf)
 const unlink = promisify(fs.unlink)
 const bbGlob = promisify(glob)
 
-export const fallbackGlob = [
-  '**',
-  '!.build/',
-  '!.build/**/*',
-  '!manifest.json',
-  '!render/',
-  '!render/**/*',
-  '!node_modules/',
-  '!node_modules/**/*',
-]
-
-export function joinWithBuild (root) {
-  return path.join(root, '.build/')
-}
-
 export function listLocalFiles (root) {
-  return bbGlob('{.build/**/*,manifest.json}', {
+  return bbGlob('{*/**,manifest.json}', {
```

Comment:
@firstdoit This glob pattern only looks for folders on the root and the manifest file?